### ISSUE TITLE
Need to better educate users about sidewalk design

### ISSUE THREAD

**Issue by jonfroehlich** [ProjectSidewalk/SidewalkWebpage#2685]
There are opportunities for us to better educate our users about sidewalk design + accessibility, including how wide sidewalks _can_ have sidewalk furniture on curbside as long as there is a sufficiently wide pedestrian zone. Could do this in our interactive tutorial, during mission screens, and in our labeling guide.

Some inspirational graphics:
![image](https://user-images.githubusercontent.com/1621749/135942113-3fda5ca1-875a-4ffc-ab55-228bc8531c4f.png)

![image](https://user-images.githubusercontent.com/1621749/135942234-3424055b-16e2-4208-b48f-5e80b059fb96.png)
from https://montgomeryplanning.org/planning/communities/midcounty/rock-spring-and-white-flint-2-design-guidelines/

Similarly, can better show/teach users about why sidewalk cross slopes matter:
![image](https://user-images.githubusercontent.com/1621749/135942408-33eefb50-b4bd-4a72-9292-38265d50c257.png)
from https://nacto.org/docs/usdg/accessible_sidewalks_and_street_crossings_boodlal.pdf

**Comment by jonfroehlich** [comment]
Here are some examples of *incorrect* obstacle labels. These objects are not in the pedestrian pathway and a wheelchair rider has plenty of room to pass. We should be able to inform our users about this via some nice graphical depictions.

![image](https://user-images.githubusercontent.com/1621749/136035404-9fdb1461-eded-4d52-b4af-35db576e7c52.png)

![image](https://user-images.githubusercontent.com/1621749/136035431-0f6b3e21-d9db-4501-a449-01266a764db1.png)

**Comment by jonfroehlich** [comment]
I personally think we could add in stuff like this (but our own sketch) into the mission screens (and other places) to help. What do you think @isavin12?

![image](https://user-images.githubusercontent.com/1621749/139963821-6ac1d7ad-a77c-41cd-a2eb-a94c17393766.png)

**Comment by isavin12** [comment]
Agreed, and from looking above I think it would be most effective to communicate this info in perspective/axon view because that's how our users will be experiencing these issues while mapping. I think this would be handy to have while mapping in a "field guide" tab like in Zooniverse:

![image](https://user-images.githubusercontent.com/56325998/140015918-3838bade-2b4d-4984-b92e-6ae5c65964fc.png)

![image](https://user-images.githubusercontent.com/56325998/140015938-09387ae6-9495-49fa-af1f-be64751dc176.png)

**Comment by jonfroehlich** [comment]
Thanks @isavin12. The key question, to me, is _where_ do we provide this info. We know that our users don't typically see or click on additional screens for more info. So, I think we have to train them in traditional flows like mission screens...

**Comment by jonfroehlich** [comment]
This [guide by the Brazilian government](https://prefeitura.pbh.gov.br/sites/default/files/estrutura-de-governo/politica-urbana/2020/pss_03prj_cartilha_r18_site.pdf) (Portuguese) also has some nice graphics.

![image](https://user-images.githubusercontent.com/1621749/144843892-a94e0638-a614-4f3e-bc12-d65d92cf301f.png)

![image](https://user-images.githubusercontent.com/1621749/144843706-eacf15fb-b28d-42d7-af6c-34beeef164f9.png)

![image](https://user-images.githubusercontent.com/1621749/144843743-da7aa601-1c02-499c-919f-c9e253eca992.png)

![image](https://user-images.githubusercontent.com/1621749/144843780-c64278a8-2ac5-4928-add4-37492d041f0e.png)

**Comment by jonfroehlich** [comment]
I've attached some feedback from Yochai's UIC undergrad class that used Project Sidewalk indicating that user education continues to be an important design focus for us

[UIC_Class_Feedback.docx](https://github.com/ProjectSidewalk/SidewalkWebpage/files/7864318/UIC_Class_Feedback.docx)
.

**Comment by jonfroehlich** [comment]
Here is another helpful annotated image about improvements to a sidewalk and curb ramp:
![image](https://github.com/ProjectSidewalk/SidewalkWebpage/assets/1621749/e09a5cd2-a855-44dd-bdf2-1961437c1c91)

Source: http://www.albanystrollroll.org/curb-ramp-redesign-success/

**Comment by jonfroehlich** [comment]
And this pocket guide is really good: https://www.dot.state.pa.us/public/Bureaus/design/ADA/PocketGuide.pdf

I think we've previously shared it in our group.
